### ISSUE TITLE
Decaying learning rate with Adam optimizer

### ISSUE THREAD

**Issue by vinaykumar2491** [pytorch/pytorch#12478]
Hi,
I'm trying to decay the learning rate using `optim.lr_scheduler.ExponentialLR()` with `optim.Adam()` optimizer.

My loss suddenly starts increasing.
![loss_vocals_decay_comparision](https://user-images.githubusercontent.com/12234745/46663142-f0ea5c00-cbda-11e8-9bfb-f0ff624edd0a.png)

What should I do for a better learning?

**Comment by pskrunner14** [comment]
@vinaykumar2491 what value of `gamma` are you using for the scheduler? Also is this training loss or validation loss?

**Comment by soumith** [comment]
https://discuss.pytorch.org

**Comment by cherepanovic** [comment]
any updates regarding this issue?
